server/tests/batch_comparison/test_batch_inventory.py

```diff
@@ -1141,13 +1155,50 @@ def test_batch_inventory_wrong_tabulator_status_file(
     rv = client.get(
         f"/api/election/{election_id}/jurisdiction/{jurisdiction_ids[0]}/batch-inventory/tabulator-status"
     )
-    tabulator_status = json.loads(rv.data)
-    assert tabulator_status["processing"]["status"] == ProcessingStatus.ERRORED
-    assert (
-        tabulator_status["processing"]["error"]
-        == 'We could not parse this file. Please make sure you upload the plain XML version of the tabulator status report. The file name should end in ".xml" and should not contain the words "To Excel".'
+    compare_json(
+        json.loads(rv.data),
+        {
+            "file": {"name": "tabulator-status.xml", "uploadedAt": assert_is_date},
+            "processing": {
+                "status": ProcessingStatus.PROCESSED,
+                "startedAt": assert_is_date,
+                "completedAt": assert_is_date,
+                "error": None,
+            },
+        },
+    )
```

Comment:
Should we test that the data was parsed correctly? I think in the happy path test above we confirm this by downloading the batch inventory worksheet

Reply:
Good call: https://github.com/votingworks/arlo/pull/2018/commits/abddab989b1569c29fd6d0404c4905b926c11fea#diff-6c78d492863efe745b0b5db8818551b44a2e7d9367ca6ee77b0b527be3a3e038R1172